online_store/gt_sports.css

```diff
@@ -0,0 +1,32 @@
+#h2title {
+color: black;
+background-color: yellowgreen;
+width: 100px;
+height: 50px;
+}
+
+#h2title2 {
+    color:yellowgreen;
+    background-color: black;
+    width: 100px;
+    height: 50px;
+}
+
+.column {
+        float: left;
```

Comment:
in general it is best to avoid floats if humanly possible. we have many other tools to style with like grid and flex box

Reply:
Travis, I looked into the grid and my first attempt worked well. Then I decided to make nested grids within more sub-grids within a grid and changed javascript accordingly. It stopped working and since then I can't figure out where my problem is. Since I've changed the document almost completely now, I will upload a new one with a new 'pr'.